x-pack/filebeat/module/coredns/kubernetes/test/coredns-json.log-expected.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "@timestamp": "2019-02-12T00:27:28.903Z",
+        "coredns.dnssec_ok": "false",
```

Comment:
Yes, it should. Although when the type is set to Boolean, Elasticsearch treats "false" exactly the same as false.

Reply:
Yeah the issue this will cause are for consumers of the raw documents, however (e.g. API access). They will always have to cast to boolean. Although since this isn't an ECS field, I guess it's less worrying for now.